packer/build_ova.sh

```diff
@@ -121,9 +127,12 @@ postProcess(){
   createChecksum
   gpgSigning
   chmod a=r "$OVA"*
-  if [ $TARGET_DIR != `pwd` ]; then
+  set +e
```

Comment:
can you add comment here why set +e?
because it's possible that "TARGET_DIR can OUTPUT_PATH are the same location, and the mv will fail"

Reply:
no~ because when build ova there's no vmdk files